lib/sycamore/sycamore/connectors/elasticsearch/elasticsearch_writer.py

```diff
@@ -88,7 +89,8 @@ def bulk_action_generator():
                         "_index": target_params.index_name,
                         "_id": r.doc_id,
                         "properties": r.properties,
-                        "embeddings": r.embeddings,
+                        "embeddings": r.embedding,
```

Comment:
hey can we use embedding (singular) in the db as well?

Reply:
Done.